packages/react-search-ui-views/src/PagingInfo.js

```diff
@@ -10,7 +10,12 @@ function PagingInfo({ className, end, searchTerm, start, totalResults }) {
       <strong>
         {start} - {end}
       </strong>{" "}
-      out of <strong>{totalResults}</strong> for: <em>{searchTerm}</em>
+      out of <strong>{totalResults}</strong>{" "}
```

Comment:
Is it possible to move the `{" "}` to line 16 instead (before the '`for`)? Ideally the extra space should only show up if needed if the word "for" appears. If Prettier gives you issue with that though, then no worries.

Reply:
I've made this change, prettier didn't throw any errors. I should've thought about this. Thanks for pointing it out ~!